src/backend/distributed/sql/udfs/citus_check_connection_to_node/11.0-1.sql

```diff
@@ -0,0 +1,11 @@
+CREATE FUNCTION pg_catalog.citus_check_connection_to_node (
+    nodename text,
+    nodeport integer DEFAULT 5432)
+    RETURNS bool
+    LANGUAGE C
+    STRICT
+    AS 'MODULE_PATHNAME', $$citus_check_connection_to_node$$;
+
+COMMENT ON FUNCTION pg_catalog.citus_check_connection_to_node (
+    nodename text, nodeport integer)
+    IS 'checks connection to another node';
```

Comment:
We may want to remove `another` word from here as well.

Reply:
Done.